docs/.vitepress/theme/components/home/HomeHeroIconsCard.vue

```diff
@@ -143,7 +143,7 @@ onBeforeUnmount(() => {
   .search-box {
     top: unset;
     bottom: -24px;
-    left: -24px;
+    /* left: -24px; */
```

Comment:
```suggestion
    left: -24px;
    width: calc(100% + 48px);
```
![image](https://github.com/lucide-icons/lucide/assets/25524993/2d31b189-8788-4f39-a6d2-4cfb537ce59f)